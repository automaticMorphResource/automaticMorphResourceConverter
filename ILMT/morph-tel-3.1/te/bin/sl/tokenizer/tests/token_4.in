గ్రేట్ బారియర్ రీఫ్... 
