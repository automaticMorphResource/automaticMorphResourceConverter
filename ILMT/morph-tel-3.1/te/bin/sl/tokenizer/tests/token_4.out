<Sentence id="1">
1	గ్రేట్	unk
2	బారియర్	unk
3	రీఫ్...	unk
</Sentence>

